verify/install-service

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -eufx
+
+SECRETS=/opt/cockpit-verify-secrets
+IMAGES=/opt/cockpit-verify-images
+
+echo "options kvm-intel nested=1" > /etc/modprobe.d/kvm-intel.conf
+echo "options kvm-amd nested=1" > /etc/modprobe.d/kvm-amd.conf
+( rmmod kvm-intel && modprobe kvm-intel ) || ( rmmod kvm-amd && modprobe kvm-amd )
```

Comment:
To avoid trying to run this in an environment without kernel modules, like a chroot or a custom kernel? Not a biggie, and can be changed later if it becomes an issue, so ok.